randovania/gui/multiplayer_session_window.py

```diff
@@ -778,6 +779,20 @@ async def _should_overwrite_presets(self, parameters: GeneratorParameters, perma
 
         return True
 
+    def _check_for_unsupported_games(self, presets: list[Preset]) -> list[str]:
+        unsupported_games = sorted(
+            {
+                preset.game.data.long_name
+                for preset in presets
+                if not preset.game.data.defaults_available_in_game_sessions
```

Comment:
One thing which I just realized now:
Should this use the session's `allowed_games` ? It should only matter in a rare edge case. It would require to have an additional game in that list and only added it via permalink / rdvgame instead of creating it "normally" (because the latter would not work if it has not the `defaults_available_in_game_sessions` set to true)

Reply:
Would be good, yeah. Can leave a TODO, but I bet `self` has the session data which has the list?

Reply:
I don't like adding TODOs. I changed it. Had to change some tests because they now require to have `allowed_games` in the mock and provide something sortable for the `long_name`.
If it's fine, feel free to hit the merge button.

(Funny that the CHANGELOG had no conflict and was added to the old release which I saw last minute).